lib/src/objdetect/objdetect_async.dart

```diff
@@ -0,0 +1,665 @@
+library cv;
+
+import 'dart:ffi' as ffi;
+import 'dart:typed_data';
+import 'package:ffi/ffi.dart';
+import '../core/base.dart';
+import '../core/mat.dart';
+import '../core/point.dart';
+import '../core/rect.dart';
+import '../core/size.dart';
+import '../core/vec.dart';
+import '../opencv.g.dart' as cvg;
+
+class CascadeClassifier extends CvStruct<cvg.CascadeClassifier> {
+  CascadeClassifier._(cvg.CascadeClassifierPtr ptr, [bool attach = true])
+      : super.fromPointer(ptr) {
+    if (attach) {
+      finalizer.attach(this, ptr.cast(), detach: this);
+    }
+  }
+
+  factory CascadeClassifier.empty() {
+    final p = calloc<cvg.CascadeClassifier>();
+    cvRun(() => CFFI.CascadeClassifier_New(p));
+    return CascadeClassifier._(p);
+  }
+
+  factory CascadeClassifier.fromFile(String filename) {
+    final p = calloc<cvg.CascadeClassifier>();
+    final cp = filename.toNativeUtf8().cast<ffi.Char>();
+    cvRun(() => CFFI.CascadeClassifier_NewFromFile(cp, p));
+    calloc.free(cp);
+    return CascadeClassifier._(p);
+  }
+
+  Future<bool> loadAsync(String name) async {
+    final cname = name.toNativeUtf8().cast<ffi.Char>();
+    final rval = cvRunAsync<bool>((callback) => CFFI.CascadeClassifier_Load_Async(ref, cname, callback), (c, p) {
```

Comment:
Here the error is because no `CascadeClassifier_Load_Async` found, check it in `objdetect_async.h`, no declaration found.